src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -926,7 +927,9 @@ def __call__(
         control_guidance_start: Union[float, List[float]] = 0.0,
         control_guidance_end: Union[float, List[float]] = 1.0,
         clip_skip: Optional[int] = None,
-        callback_on_step_end: Optional[Callable[[int, int, Dict], None]] = None,
+        callback_on_step_end: Optional[
+            Union[Callable[[int, int, Dict], None], PipelineCallback, MultiPipelineCallbacks]
+        ] = None,
```

Comment:
Should the docstrings also reflect the type change? 

(applies here and elsewhere)